unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/LoadingScreen/Tests/DCL.LoadingScreen.Tests.asmdef

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "DCL.LoadingScreen.Tests",
+    "rootNamespace": "",
+    "references": ["GUID:410d113a616fe7a46bdbf47941da22a4","GUID:97d8897529779cb49bebd400c7f402a6"],
+    "includePlatforms": [],
```

Comment:
Please include "Editor" only, otherwise these tests will be for "Play Mode" that is not needed

Reply:
Done! Lets see how we can add the EditMode tests into the CI. Our current flow will ignore these tests otherwise